index.js

```diff
@@ -251,6 +251,32 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		const killResult = originalKill(signal);
+		if (signal === undefined ||
+			(typeof signal === 'string' && signal.toUpperCase() === 'SIGTERM') ||
+			(typeof signal === 'number' && signal === os.constants.signals.SIGTERM)) {
+			const retry = !(options !== undefined && options.retry === false);
+
+			if (retry && killResult) {
+				const retryAfter = options !== undefined &&
+				['number', 'string'].includes(typeof options.retryAfter) ?
```

Comment:
Why does it accept `string`? That makes no sense to me.

Reply:
Fixed: a09b4a35609b4423ce14d5424539cd6e6c39f13f